pkg/mongoproxy/plugins/schema/type_test.go

```diff
@@ -25,13 +25,13 @@ var (
 		{DB: "testdb", Collection: "nonrequire", In: bson.D{}, Err: false},
 		{DB: "testdb", Collection: "testcollection", In: bson.D{}, Err: true},
 		// Check string array
-		{DB: "testdb", Collection: "testcollection", In: bson.D{{"friends", bson.A{"bob", "alice"}}}, Err: false},
+		{DB: "testdb", Collection: "testcollection", In: bson.D{{"friends", bson.A{"bob", "alice"}}, {"name", "tom"}}, Err: false},
 		// Check string array with wrong type
 		{DB: "testdb", Collection: "testcollection", In: bson.D{{"friends", bson.A{"bob", 1}}}, Err: true},
 		// Check int array
 		{DB: "testdb", Collection: "nonrequire", In: bson.D{{"luckynumbers", bson.A{666, 888}}}, Err: false},
 		// Check int array with wrong type
-		{DB: "testdb", Collection: "nonrequire", In: bson.D{{"luckynumbers", bson.A{666, "888"}}}, Err: true},
+		{DB: "testdb", Collection: "nonrequire", In: bson.D{{"luckynumbers", bson.A{666, "888"}}}, Err: false},
```

Comment:
why this change?

Reply:
since this field is none required, we should not validate it.